fastlane/docs/Guide.md

```diff
@@ -125,16 +118,22 @@ error do |lane, exception|
 end
 ```
 
-You can already try running it, put a line (`say "It works"`) to the `:inhouse` lane and run
+You can already try running `fastlane`, add this to your `Fastfile`
+```ruby
+lane :example do
+  say "It works"
+end
+```
+
+And then run the following from your Terminal
 
-    fastlane inhouse
+    fastlane example
 
 You should hear your computer speaking to you, which is great!
 
 A list of available actions can be found in the [Actions documentation](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md).
 
-Automating the deployment process is a great next step. You should use `increment_build_number` when you want to upload builds to iTunes Connect ([Activate incrementing build numbers](https://developer.apple.com/library/ios/qa/qa1827/_index.html)).
-
+Automating the deployment process is a great next step. You should use the `increment_build_number` action when you want to upload builds to iTunes Connect ([Activate incrementing build numbers](https://developer.apple.com/library/ios/qa/qa1827/_index.html)).
```

Comment:
Do we want to make it app specific and not iOS specific since this works for Android right?